### ISSUE TITLE
Unrecognized options "time-limit" and "failure-limit"

### ISSUE THREAD

**Issue by nkrewcun** [win32service/Win32ServiceBundle#15]
I tried setting up the time-limit or failure-limit in my yaml config file like so:
```yaml
win32_service:
  project_code: project
  messenger:
    -
      receivers: [aasync]
      displayed_name: Messenger Async worker %d
      limit: 500
      time-limit: 300
      failure-limit: 300

```

But encountered the following error:
> Unrecognized options "time_limit, failure_limit" under "win32service.messenger.0". Did you mean "time-limit", "failure-limit"?

I also tried putting single quotes around the options but still get the same error.

The Symfony ArrayNode seems to be normalizing the options by changing the - to an _ and it prevents the use of these two options.

Could you fix this please?

**Comment by macintoshplus** [comment]
See #16

**Comment by macintoshplus** [comment]
Fixed in release v2.0.1
